mozilla_schema_generator/main_ping.py

```diff
@@ -14,6 +14,11 @@
 
 class MainPing(GenericPing):
 
+    # Only includes probes that have been available at some point past
+    # this version.
+    # ONLY DECREMENT, or the schema will change in an incompatible way!
+    MIN_FX_VERSION = 60
```

Comment:
Any particular reason to choose Firefox 60?

Reply:
I chose this because that is what ESR is on right now. Though we are pushing to 68 very soon.